simple_commands.py

```diff
@@ -126,10 +126,11 @@ def stats(bot, update):
         )
 
         n = us.first_places
+        m = int((us.games_played / us.first_places) * 100) if us.first_places else 0
```

Comment:
Just a slight change, using `round` instead of `int` 

```python
>>> int(0.8)
0
>>> round(0.8)
1
```

```suggestion
        m = round((us.games_played / us.first_places) * 100) if us.first_places else 0
```